Library/Homebrew/cxxstdlib.rb

```diff
@@ -1,23 +1,30 @@
-# typed: true # rubocop:todo Sorbet/StrictSigil
+# typed: strong
```

Comment:
Why `strong` and not `strict`?

Reply:
Mainly since it is possible to increase strictness here. We do have a couple files elevated to `strong` (i.e. no-`T.untyped`).

Can also just use `typed: strict`. Shouldn't have a major impact as Sorbet has a lot of `T.untyped` usage (e.g. JSON) so won't be able to use `typed: strong` in most of `brew`.

Reply:
Looks like it's still in beta: https://sorbet.org/docs/strong

Not opposed to it, they shout less loudly about using it than they used to!

Reply:
I use (and require) it in ci-orchestrator except the Sinatra file and an Octokit wrapper.

It makes it significantly less likely to get TypeErrors at runtime by banning `T.untyped`, though most third-party gems are `T.untyped` so is often extremely difficult to intergate into a codebase.

For CI Orchestartor I had to send various PRs upstream to fix various stdlib `T.untyped` signatures. But it was worth it as it gives really high confidence the code is at least valid.